### PR TITLE
Use dotbot

### PR DESCRIPTION
**Pull request by OxSon**
I restructured your project to use dotbot like I suggested and added a readme with set up instructions.

### CODE REVIEW

**Comment by OxSon** [comment]
i wasn't sure how vscode conf files are supposed to be structured so i assumed they go in a folder in the home directory named `.vscode`. if this isn't right you'll have to fix the conf file.

i also wasnt sure if you use `~/.config` as your config file home like a sane person so I just assumed you're not a mad man. If you are, you'll have to fix the conf file and directory structure to your match your absurd choices

**Comment by OxSon** [comment]
I did a massive overhaul and added a plugin I adapted from someone else. With my PR dotbot should now be capable of a complete bootstrapping of your system with only `yay` as an initial dependency outside of the repo itself. You're welcome bruh bruh

**Comment by OxSon** [comment]
Refer to [my readme](https://github.com/OxSon/.dotfiles/blob/master/README.md) or call me for questions/problems/answers

File: README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#### 3. Install 
+  `./install`
+  * Note: at this point you may want to edit the config file at `.dotfiles/default.conf.yaml`. Please refer to the excellent project documentation available at [the dotbot project page](https://github.com/anishathalye/dotbot).
+
+
+---
+
+❌ ***Known issues***
+
+user tarballz is super stinky
```
Comment (tarballz) [inline README.md:23]: Nice.

**Review by tarballz — APPROVED** [review]
nice work butthead.

**Pull request merged**